host/sgx/windows/enter.asm

```diff
@@ -74,22 +83,15 @@ NESTED_ENTRY oe_enter, _TEXT$00
     mov rax, [rbp+64]
     mov ENCLAVE, rax
 
-    ;;Save the current context
-
-    ;;Save the SSE status and control flags
-    stmxcsr MXCSR
-	
-    ;; Save registers:
-    push rbx
-    push rdi
-    push rsi
-    push r12
-    push r13
-    push r14
-    push r15
+    ;; Store the host context on the host stack
+    mov HOST_CONTEXT, rsp
```

Comment:
This line is hard to read. 
The comment in line 53 says `64 bytes for all preserved registers in HOST_CONTEXT` but this line just stores the current rsp value in HOST_CONTEXT

Reply:
Conceptually, the `HOST_CONTEXT` is just a pointer to the start of that block of information used in host_context.asm, which includes the 8 registers and fxsave memory image and is just sequentially pushed onto the stack. 

What would you suggest to clarify this? Is the confusion around "in" `HOST_CONTEXT`?

Reply:
Okay, I've updated the comments around the definition of `PARAMS_SPACE` and `HOST_CONTEXT`, let me know if that still doesn't help.

Reply:
Ok. Thanks for clarifying.